infra/ansible/inventory.yml

```diff
@@ -178,6 +178,8 @@ all:
           # different languages depending on the day of the week.
           tournesol_twitterbot_schedule: "*-*-* 18:03:00 Europe/Zurich" # daily.
 
+          tournesol_yt_playlists_schedule: "*-*-* 07:15:00" # daily
```

Comment:
I would put even earlier so people have fresh videos for their morning commute 😀

Reply:
This was to be launched after the ML run, running approximately between 06:20 et 06:50 UTC

That's not very important,let's move it to 05:15 UTC

Reply:
Done in acf3e43a